elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/ElasticSearchAdapterTest.java

```diff
@@ -462,6 +462,14 @@ public static void setupInstance() throws Exception {
         .explainContains(explain);
   }
 
+  /**
+   * Range query is not supported currently.
+   * @see <a href="https://issues.apache.org/jira/browse/CALCITE-4645">[CALCITE-4645]
+   * In Elasticsearch adapter, a range predicate should be translated to a range query</a>
+   * <p>Description about translating search call to range query
+   * {@link org.apache.calcite.adapter.elasticsearch.PredicateAnalyzer.Visitor#canBeTranslatedToTermsQuery(RexCall)}
+   */
+  @Disabled
```

Comment:
I think it is better to create an entry in https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/util/Bug.java class and use that value to skip/disable the test.

Reply:
Hi, @zabetak .I didn't know such an entry for Bug label before. 
I've added those two bugs there, thanks for your reminder!